addons/web_editor/static/src/js/common/grid_layout_utils.js

```diff
@@ -189,14 +193,17 @@ function _placeColumns(columnEls, rowSize, rowGap, columnSize, columnGap) {
         const columnEnd = columnStart + columnSpan;
 
         // Vertical placement.
-        const columnTop = isImageColumn ? imageEl.offsetTop : columnEl.offsetTop;
+        const borderTop = parseFloat(style.borderTop);
+        const columnTop = isImageColumn && !borderTop ? imageEl.offsetTop : columnEl.offsetTop;
         // Getting the top and bottom paddings and computing the row offset.
         const paddingTop = parseFloat(style.paddingTop);
         const paddingBottom = parseFloat(style.paddingBottom);
         const rowOffsetTop = Math.floor((paddingTop + rowGap) / (rowSize + rowGap));
         // Getting the height of the column.
-        const height = isImageColumn ? parseFloat(imageEl.scrollHeight)
+        let height = isImageColumn ? parseFloat(imageEl.scrollHeight)
             : parseFloat(columnEl.scrollHeight) - (allBackgroundColor ? 0 : paddingTop + paddingBottom);
+        const borderY = borderTop + parseFloat(style.borderBottom);
```

Comment:
Same here why not using offsetHeight ?
If not possible to use it, just a suggestion => add the comment to explain why. So that other devs know they must keep `scrollwidth` and `scrollHeight` if they make a change (cause it's not obvious)